tests/e2e/e2e_test.go

```diff
@@ -211,3 +212,33 @@ func (s *IntegrationTestSuite) TestStateSync() {
 	err = chain.RemoveNode(stateSynchingNode.Name)
 	s.NoError(err)
 }
+
+func (s *IntegrationTestSuite) TestExpeditedProposals() {
+	if !s.skipUpgrade {
+		s.T().Skip("this can be re-enabled post v12")
+	}
+
+	var wg sync.WaitGroup
```

Comment:
What is the purpose of using both a channel and a wait group? Waitgroup seems redundant since we would block on a channel until it gives us a value anyways

Reply:
I actually wasn't aware that channels block until a value is returned. I tested it just to make sure and that is the case, thanks!